index.html

```diff
@@ -1,144 +1,99 @@
 <!DOCTYPE html>
 <html>
-  <head>

```

Comment:
Viewing current snapshot, I see a respec validation error: 

"Found linkless element for 'performance' with text 'ensure name is valid' but no matching <dfn>."

Does this algorithm still exist / is needed?

Reply:
Oops, forgot to remove this line when updating the spec to fix #36.